tests/behaviour/mod.rs

```diff
@@ -35,10 +42,17 @@ pub struct Context {
     pub connection: Connection,
     pub databases: DatabaseManager,
     pub session_trackers: Vec<SessionTracker>,
+    pub things: HashMap<String, Option<Thing>>,
 }
 
 impl Context {
     async fn test(glob: &'static str) -> bool {
+        let default_panic = std::panic::take_hook();
+        std::panic::set_hook(Box::new(move |info| {
+            default_panic(info);
+            std::process::exit(1);
+        }));
```

Comment:
This ensures that panics in the background thread aborts the process instead of it hanging. This is an issue in the way panics are handled in the background runtime, a fix for which is out of scope for this PR.

Reply:
Yes.